update docs for API v1beta2 (#356)
OpenFunction/openfunction.dev: content/en/docs/concepts/function_trigger.md

```yaml
apiVersion: core.openfunction.io/v1beta2
kind: Function
metadata:
  name: function-sample
spec:
  serving:
    triggers:
      http:
        port: 8080
        route:
          rules:
            - matches:
                - path:
                    type: PathPrefix
                    value: /echo
```

## Dapr Trigger

Dapr trigger is a event trigger using dapr bindings and pubsub. The function will be triggered when receive a event from dapr bindings or pubsub.

To define a dapr trigger, you can follow this:

```yaml
apiVersion: core.openfunction.io/v1beta2
kind: Function
metadata:
  name: logs-async-handler
  namespace: default
spec:
  serving:
    bindings:
      kafka-receiver:
        metadata:
          - name: brokers
            value: kafka-server-kafka-brokers:9092
          - name: authRequired
            value: "false"
          - name: publishTopic
            value: logs
          - name: topics
            value: logs
          - name: consumerGroup
            value: logs-handler
        type: bindings.kafka
        version: v1
    triggers:
      dapr:
        - name: kafka-receiver
          type: bindings.kafka
```

## Function Inputs

> Function input is a component that function can get data from, now only support dapr state store.

Reviewer: ```suggestion
`Input` is where a function can get extra input data from, `Dapr State Stores` is supported as `Input` currently.
```

> Function input is a component that function can get data from, now only support dapr state store.

Reviewer: ```suggestion
`Input` is where a function can get extra input data from, `Dapr State Stores` is supported as `Input` currently.
```

> To define a input, you can follow this:

Reviewer: ```suggestion
You can define function input like this:
```
